backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -615,6 +616,20 @@ def open_message_stream(self, context, request: TracimRequest, hapic_data=None)
         stream_opened_event = ":Tracim Live Messages for user {}\n\nevent: stream-open\ndata:\n\n".format(
             str(request.candidate_user.user_id)
         )
+
+        if "after_event_id" in hapic_data.query:
```

Comment:
You don't need the tests here, provided you have a schema-defined default value for after_event_id.